### DISCUSSION TITLE
unable to call endpoint - error 404

### DISCUSSION THREAD

**Discussion by teafmartins** [serenity-is/Serenity#5495] [Help]
Hello,

I'm tryng to call endpoint directly in browser 
http://localhost/myproject/Services/Company/List
and get error 404:
[HttpException]: No matching action was found on controller &#39;myproject.Endpoints.CompanyController&#39;. This can happen when a controller uses RouteAttribute for routing, but no action on that controller matches the request.
   em System.Web.Mvc.Controller.HandleUnknownAction(String actionName)

Northwind customer list works fine!
http://localhost/myproject/Services/Northwind/Customer/List 

Any help? Thanks

**Comment by VictorTomaili** [comment]
you forget module name in your url

**Reply by teafmartins** [reply]
added, but the problem contnue? any other  clue? 
thx

**Reply by VictorTomaili** [reply]
there is no reason if you generate this module from sergen it's must work. Check it again.

**Reply by VictorTomaili** [reply]
Also you can access service from Northwind.CustomerService.List 
It's must generated for your services too. {namespace}.{module}.{controllername}Service.{method} pattern should work.

**Reply by teafmartins** [reply]
Not working, that's strange.
Frist time I generated code with sergen without module name... after that I manually added a module name. 
Can this be the problem?

**Reply by teafmartins** [reply]
Found it!

Endpoint.ts has "post" attribute... browser is get :)

        [HttpPost]
        public ListResponse<MyRow> List(IDbConnection connection, ListRequest request)
        {
            return new MyRepository().List(connection, request);
        }
